cfe_internal/enterprise/federation/federation.cf

```diff
@@ -345,10 +356,13 @@ bundle agent clean_when_off
 
   commands:
       # Oh, the humanity! Where for art thou databases: promises for psql!
-      `$(sys.bindir)/psql cfsettings --quiet --command "TRUNCATE TABLE remote_hubs"` -> { "ENT-7233" }
-        if => isgreaterthan( "$(remote_hubs_table_row_count)", "0" );
-      `$(sys.bindir)/psql cfsettings --quiet --command "TRUNCATE TABLE federated_reporting_settings"` -> { "ENT-7233" }
-        if => isgreaterthan( "$(federated_reporting_settings_table_row_count)", "0" );
+      `$(sys.bindir)/psql cfsettings --quiet --command "TRUNCATE TABLE remote_hubs"` -> { "ENT-7233", "CFE-951" }
```

Comment:
Again, you aren't containing as `cfpostgres` user and I don't think you need to as the root postgres role/user can do these operations because they are given "superuser" privileges as mentioned in the above buildscripts post install script content.